cedar-policy-validator/src/schema/raw_name.rs

```diff
@@ -105,3 +88,9 @@ impl std::str::FromStr for RawName {
         Name::from_str(s).map(RawName)
     }
 }
+
+impl From<RawName> for Name {
+    fn from(value: RawName) -> Self {
+        value.0
+    }
+}
```

Comment:
I'd prefer we don't have this impl, because we don't want folks accidentally treating a `RawName` as a fully-qualified `Name` without being intentional about whether it needs to be qualified.  (That's the reason for the `RawName` wrapper in the first place.)  I'd prefer we remove this and instead use `raw_name.qualify_with(None)` wherever it was needed, which is more explicit about claiming that the name is indeed in the empty namespace.